README.md

```diff
@@ -16,8 +16,36 @@ cap config:check:setup_files_exists_local        # Check Setup files exists in L
 cap config:check:upload_setup_files              # Check Setup files are exists, if not upload files
 
 cap migrations:check                             # check if migrations should be run
+
+cap deploy:compile_assets                        # Download or compile assets
```

Comment:
@tmking If we're compiling assets on the server or application instance, can we ensure that both Node and Yarn are set to the correct versions on that instance? In a recent update, we didn't verify this, assuming that since asset compilation happens in CI, the app instance wouldn't need these libraries. However, if we ever compile assets on the app instance itself, having the correct versions of Node and Yarn becomes essential.

Reply:
Node and yarn are installed when the AMI is created. The version numbers are specified in network-for-good/nfg-infrastructure: https://github.com/network-for-good/nfg-infrastructure/blob/95521e0436512d9c37b9a7cc1ba1ef4f94a18be8/ansible/playbooks/vars/dm.yml#L10. 

If the node or yarn versions change, you can open a BDO ticket to request that the versions are updated in nfg-infrastructure. Or you can open a PR in that repo and tag me on it.

Reply:
Assets could be compiled inline on app instances if there were no assets for that build present on S3. In practice, this rarely ever happens now that deployments have been moved into CircleCI, where the assets are compiled before the deploy happens.

Reply:
@tmking thanks that makes sense, I wonder if we ever want to pull the plug and just make that an error case somehow. I don't think that's related to this though so we can take that discussion elsewhere.